### PR TITLE
Fixing #72, using helm uninstall instead of helm delete and aded code for replace flag

### PR DESCRIPTION
**Pull request by dicksol**
Replaced helm delete by helm uninstall as described here: https://helm.sh/docs/helm/helm_delete/
Also implemented code for the replace flag.

### CODE REVIEW

**Comment by Typositoire** [comment]
Concourse: Version bump did not happen. v1.20.0 is the same as v1.20.0

**Comment by rws-deltares** [comment]
@Typositoire, do you expect anything from my side to perform the merge?

**Comment by dicksol** [comment]
@Typositoire, do you expect anything from my side to have the code change merged.

File: assets/common.sh [hunk 1]
```diff
@@ -116,7 +116,7 @@ setup_helm() {
     local release
     for release in $(helm ls -aq --namespace $namespace )
     do
-      helm delete --purge "$release" --namespace $namespace
+      helm uninstall --purge "$release" --namespace $namespace
```
Comment (Typositoire) [inline assets/common.sh:119]: You can remove purge, this is no longer a thing (It was kept when moved from v2)

**Review by Typositoire — CHANGES_REQUESTED** [review]
Please bump the minor version also! And sooooo sorry for the delay :(

**Comment by dicksol** [comment]
@Typositoire, processed your remarked. Please merge if okay. Tx.

**Comment by dicksol** [comment]
Lovely. Tx.

**Comment by Typositoire** [comment]
Look for a new image in ~10 minutes. (Concourse is doing his job)

**Comment by Typositoire** [comment]
https://github.com/Typositoire/concourse-helm3-resource/pkgs/container/concourse-helm3-resource/13693660?tag=v1.21.0

**Comment by dicksol** [comment]
#71, also solved now.

**Comment by dicksolrws** [comment]
And using it:
Pulling ghcr.io/typositoire/concourse-helm3-resource@sha256:59a179531c97e066a4527948a1336d535768c7a22e46e7e3f2a9f8220bc0b537...

ghcr.io/typositoire/concourse-helm3-resource@sha256:59a179531c97e066a4527948a1336d535768c7a22e46e7e3f2a9f8220bc0b537: Pulling from typositoire/concourse-helm3-resource

**Pull request merged**